packages/server/src/utils/workflow.ts

```diff
@@ -1,127 +1,66 @@
 import { LlamaIndexAdapter, StreamData, type JSONValue } from "ai";
 import type {
-  AgentInputData,
   ChatResponseChunk,
   EngineResponse,
   Metadata,
   NodeWithScore,
-  WorkflowEvent,
+  WorkflowEventData,
+  WorkflowStream,
 } from "llamaindex";
 import {
-  AgentStream,
-  AgentToolCall,
-  AgentToolCallResult,
+  agentStreamEvent,
+  agentToolCallEvent,
+  agentToolCallResultEvent,
```

Comment:
the workflow events are now all in `@llamaindex/workflow` - not `llamaindex`

Reply:
don't use anything workflow related from `llamaindex`!

Reply:
(we just re-export an outdated version for compatibility, see https://github.com/run-llama/LlamaIndexTS/pull/1907)